auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -371,6 +373,66 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
     return this.selectedColumns.includes(col)
   }
 
+  /** Custom sorting method to handle consolidated Nr and Affiliations data. */
+  private customSort (items, index, isDesc): any {
+    items.sort((a, b) => {
+      if (index[0] === 'lastModified') {
+        let dateA, dateB
+        if (a.lastModified) {
+          dateA = a.lastModified
+        } else dateA = a.modified
```

Comment:
better with {} for else block also. I think we are following with blocks even for single line  thoughts?

Reply:
Absolutely! Will add 👍

Reply:
Updated!